CHANGELOG.md

```diff
@@ -8,6 +8,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/) and this
 repository that is by invitation only. You will get a 404 - Not Found if you do
 not have access**
 
+## [5.14.0](https://github.com/FortAwesome/Font-Awesome/releases/tag/5.14.0) - 2020-07-15
+
+**Minor version upgrade notice: there are some backward-incompatible changes to this release. See the
+[UPGRADING.md guide](https://github.com/FortAwesome/Font-Awesome-Pro/blob/master/UPGRADING.md) for more
```

Comment:
@robmadole can I change this link to the free repo?

I always forget if the changelog should be fixed upstream or in this repo

Reply:
Yeah, feel free to. This can be freely edited.